src/DotNet/Importer.cs

```diff
@@ -261,12 +261,15 @@ public Importer(ModuleDef module, ImporterOptions options, GenericParamContext g
 			}
 		}
 
-		ITypeDefOrRef TryResolve(TypeRef tr) {
+		ITypeDefOrRef TryResolve(ITypeDefOrRef type) {
```

Comment:
I see you changed the signature of this method. However, I do not see a place where this new signature is used/necessary.